nx.json

```diff
@@ -0,0 +1,40 @@
+{
+  "extends": "nx/presets/npm.json",
+  "tasksRunnerOptions": {
+    "default": {
+      "runner": "@nrwl/nx-cloud",
+      "options": {
+        "cacheableOperations": ["build", "test", "lint"],
+        "accessToken": "ZTNjOTZlNmQtMjJlYy00NjJkLWEyM2MtNzk2OTllOWJmZmVlfHJlYWQtd3JpdGU="
```

Comment:
Should the token be hard coded here?

Reply:
Hi!

nx itself leaves it open too: https://github.com/nrwl/nx/blob/master/nx.json#L17

Reply:
Hmm, ok, but do you know if it generates any vulnerability in our project?

Reply:
Won't bring that kind of problem

Reply:
Checking [the NX doc](https://nx.app/docs/configuring-nx-cloud-cli), they are getting this value from an env. Maybe it could be a better approach